error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'

Hey!

So I'm going through the examples in the docs and the Observable counter is failing to compile using typescript.

Code for reproduction:

```javascript
// master.ts
import {spawn, Thread, Worker} from "threads"

async function start() {
    const counter = await spawn(new Worker("./workers/counter"));

    counter.subscribe((newCount: number) => console.log(`Counter incremented to:`, newCount));
}

start().catch(err => console.error(err));
```

```javascript
// workers/counter.ts
import {Observable} from "observable-fns"
import {expose} from "threads/worker"

function startCounting() {
    return new Observable(observer => {
        for (let currentCount = 1; currentCount <= 10; currentCount++) {
            observer.next(currentCount);
        }
        observer.complete();
    })
}

expose(startCounting);
```

The TS config file:

```javascript
{
  "compilerOptions": {
    /* Basic Options */
    // "incremental": true,                   /* Enable incremental compilation */
    "target": "es2018",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
    // "lib": [],                             /* Specify library files to be included in the compilation. */
    // "allowJs": true,                       /* Allow javascript files to be compiled. */
    // "checkJs": true,                       /* Report errors in .js files. */
    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
    // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
    // "sourceMap": true,                     /* Generates corresponding '.map' file. */
    // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./dist",                        /* Redirect output structure to the directory. */
     "rootDir": "./src",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
    // "composite": true,                     /* Enable project compilation */
    // "tsBuildInfoFile": "./",               /* Specify file to store incremental compilation information */
    // "removeComments": true,                /* Do not emit comments to output. */
    // "noEmit": true,                        /* Do not emit outputs. */
    // "importHelpers": true,                 /* Import emit helpers from 'tslib'. */
    // "downlevelIteration": true,            /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
    // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */

    /* Strict Type-Checking Options */
    "strict": true,                           /* Enable all strict type-checking options. */
    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
    // "strictNullChecks": true,              /* Enable strict null checks. */
    // "strictFunctionTypes": true,           /* Enable strict checking of function types. */
    // "strictBindCallApply": true,           /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
    // "strictPropertyInitialization": true,  /* Enable strict checking of property initialization in classes. */
    // "noImplicitThis": true,                /* Raise error on 'this' expressions with an implied 'any' type. */
    // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */

    /* Additional Checks */
    // "noUnusedLocals": true,                /* Report errors on unused locals. */
    // "noUnusedParameters": true,            /* Report errors on unused parameters. */
    // "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
    // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */

    /* Module Resolution Options */
     "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
    // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
    // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
    // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
    // "typeRoots": [],                       /* List of folders to include type definitions from. */
    // "types": [],                           /* Type declaration files to be included in compilation. */
    // "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
    "esModuleInterop": true,                  /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
    // "preserveSymlinks": true,              /* Do not resolve the real path of symlinks. */
    // "allowUmdGlobalAccess": true,          /* Allow accessing UMD globals from modules. */

    /* Source Map Options */
    // "sourceRoot": "",                      /* Specify the location where debugger should locate TypeScript files instead of source locations. */
    // "mapRoot": "",                         /* Specify the location where debugger should locate map files instead of generated locations. */
    // "inlineSourceMap": true,               /* Emit a single file with source maps instead of having a separate file. */
    // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */

    /* Experimental Options */
    // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
    // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */

    /* Advanced Options */
    "forceConsistentCasingInFileNames": true  /* Disallow inconsistently-cased references to the same file. */
  }
}
```

The package.json

```javascript
{
  "name": "threads-example",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "compile": "tsc -p tsconfig.json"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@types/node": "^13.7.1",
    "threads": "^1.1.0",
    "ts-node": "^8.6.2",
    "typescript": "^3.7.5"
  }
}
```

And finally, the console output from the TS compiler:

```bash
$ npm run compile
node_modules/observable-fns/dist/observable.d.ts:11:13 - error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.

11     [Symbol.observable](): Observable<T> | ObservableLike<T>;
               ~~~~~~~~~~

node_modules/observable-fns/dist/observable.d.ts:45:13 - error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.

45     [Symbol.observable]: () => this;
               ~~~~~~~~~~
```

What's also funny is when I'm running this sample using ts-node (and when I rewrite it to JS with `require` instead of `import`) then node complains that `counter.subscribe` is not a function.

```bash
$ ts-node -P tsconfig.json src/observables/master.ts 
TypeError: counter.subscribe is not a function
    at start (/home/gowg/proj/poc/azure-iot/threads-example/src/observables/master.ts:6:13)
```

I'm using NPM 6.13.7 and node 12.16.0

--- a/docs/usage-observables.md
+++ b/docs/usage-observables.md
@@ -21,7 +21,7 @@ import { spawn, Thread, Worker } from "threads"
 
 const counter = await spawn(new Worker("./workers/counter"))
 
-counter.subscribe(newCount => console.log(`Counter incremented to:`, newCount))
+counter().subscribe(newCount => console.log(`Counter incremented to:`, newCount))
 ```
 
 ```js

